BlueShift-iOS-SDK/BlueShiftConfig.h

```diff
@@ -18,6 +18,10 @@
 @property NSURL *cartPageURL;
 @property NSURL *offerPageURL;
 
+@property BOOL disablePushNotification;
```

Comment:
minor:
can we change the naming here: I think it's better to use positive connatation. 
eg: enablePushNotification
eg: enableLocationAccess
eg: enableAnalytics